docs/uax-with-aj1/Request-to-revise-UAX-50-for-the-harmonization-with-Adobe-Japan1.md

```diff
@@ -20,8 +25,9 @@ Although the technical content of UAX 50 has been fairly stable since 2015, neit
 
 CITPC would like to request the Unicode Consortium to revisit the categorization of four characters in UAX 50. Although the current categorization is sensible, its practical benefits are less significant than the adoption cost, as we see it.  If these changes are accepted, existing Japanese AJ1 fonts will become compliant with UAX 50. 
 
+## Proposal
 
-## U+2016 DOUBLE VERTICAL LINE
+### U+2016 DOUBLE VERTICAL LINE

```

Comment:
In one place you have 'vert' and another just vert.  Perhaps apply code markdown, so as to produce `vert`?

Reply:
Thank you Richard @r12a for improving the language. It now feels smoother and more confident.

Reply:
@himorin, I would appreciate it if you could incorporate the changes.